doc/multisig-tutorial.md

```diff
@@ -120,6 +115,7 @@ Once the wallets have already been created and this tutorial needs to be repeate
 
 ```bash
 for ((n=1;n<=3;n++)); do ./build/bin/bitcoin-cli -signet loadwallet "participant_${n}"; done
+./build/bin/bitcoin-cli -signet loadwallet "multisig_wallet_01"
```

Comment:
nit: you could use `./build/bin/bitcoin rpc` instead. This removes the need for `-named` elsewhere in the tutorial.